numpy/lib/shape_base.py

```diff
@@ -60,7 +60,7 @@ def apply_along_axis(func1d, axis, arr, *args, **kwargs):
     Examples
     --------
     >>> def my_func(a):
-    ...     \"\"\"Average first and last element of a 1-D array\"\"\"
```

Comment:
Not sure I agree with this change

Reply:
Should I revert that change and just remove the blank lines?

Reply:
Yeah, I think so

Reply:
okay :)